projects/vdk-plugins/vdk-jupyter/build-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,238 @@
+# Learn How to Build a Data Job through the Jupyter UI
```

Comment:
How to Develop and not build. Build has slightly different meaning often in software